plugins/dbviewer/frontend/public/javascripts/countly.views.js

```diff
@@ -464,7 +513,7 @@ app.route('/manage/db/:dbs/:collection', 'dbs', function (db, collection) {
 	this.renderWhenReady(this.dbviewerView);
 });
 
-app.route('/manage/db/:dbs/:collection/*document', 'dbs', function (db, collection, document) {
+app.route('/manage/db/:dbs/:collection/:document', 'dbs', function (db, collection, document) {
```

Comment:
We need this as splash parameter, to allow document _ids containing /, so it would not break url.
Please recheck changes before you are commiting to ensure these are your own changes and you are not overwriting changes of others